<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentDialog</class>
 <widget class="QDialog" name="StudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>396</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>299</width>
    <height>396</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>299</width>
    <height>396</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Параметри обробки</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Тест</string>
   </property>
   <widget class="QComboBox" name="cbTest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Діяльнісно-операційний</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Інформаційно-технологічний</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Когнітивний з маркетингу туризму</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Когнітивний з менеджменту</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Особистісно-творчий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Мотиваційний</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Попереднє тестування</string>
   </property>
   <widget class="QCheckBox" name="cbGroupBegin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Всі групи</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="lwGroupBegin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата тестування:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="deBegin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Поточне тестування</string>
   </property>
   <widget class="QCheckBox" name="cbGroupCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Всі групи</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="lwGroupCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата тестування:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="deCurrent">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StudentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StudentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
